public/js/office.web.js

```diff
@@ -49,6 +49,8 @@ $(() => {
   function userInMeetDecorator(user, userView) {
     const userInMeet = user.inMeet === true;
 
+    if (userInMeet) { userView.attr("title", user.name); }
```

Comment:
I tried to fix it only using CSS but I failed :(. I did a research and as it seems is related with this [line](https://github.com/ResultadosDigitais/matrix/blob/d4d8f595c2eabc8be4b6e54f13256fa26531a471/public/css/office.css#L184)